src/utils/_z-index.scss

```diff
@@ -0,0 +1,3 @@
+@function z($layer) {
```

Comment:
btw, why not `z-index`?

I really dislike the approach of making everything short... we are not saving bytes in the compilation and makes stuff harder to understand :(

Reply:
It's not about saving bytes, it's about not writing names too long, the same reason we're following emmet abbreviations for variable names: https://docs.emmet.io/cheat-sheet/. I have just followed this standard but I'm open to change it if the most of @SUI-Components/developers don't feel comfortable with it.

Reply:
Guess is a discussion for a total different than this PR so go ahead.

I just that I personally don't understand this decision at all 😢

Reply:
emmet is a global convention well know for naming of common aspects.
IMO we should not create new shorthands for specific stuff.

Reply:
I like the emmet approach. :+1:

Reply:
I personally would prefer a descriptive name, at least for functions